### PR TITLE
User sign up

### PR DESCRIPTION
**Pull request by langphil**

### CODE REVIEW

**Comment by langphil** [comment]
User can sign up for an account, edit account and delete account.

Next steps are, sign in / sign out.

**Review by dbugsy — APPROVED** [review]

**Comment by dbugsy** [comment]
:shipit: :shipit:

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (i-hardy) [inline spec/models/user_spec.rb]: This test doesn't seem to do anything?

**Review by memunyawiri — APPROVED** [review]
:shipit:

**Review by i-hardy — APPROVED** [review]
:shipit:

**Pull request merged**